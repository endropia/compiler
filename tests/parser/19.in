function a(b : integer);
var c: integer;
begin
   b := 2;
   c := b;
end;

begin end.