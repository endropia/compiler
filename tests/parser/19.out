(1, 24) ':' expected